### ISSUE TITLE
Starting a recording in Standalone player?

### ISSUE THREAD

**Issue by Cheeseless** [Unity-Technologies/GenericFrameRecorder#30]
I've got this working in the Editor, either as a tool, or in the timeline, but I can't quite figure out what to do to get it working in the standalone player. 

All I have is the same Timeline attached to MainCamera, with the Recorder Clip set the exact same way as when it worked in the editor. 

A single script detects a spacebar press to start playing the Timeline (again, working in the editor in play mode). 

However, no file is created, and no reaction seems to occur at all in the Player. Am I missing something obvious?

**Comment by jasonm-unity** [comment]
Testing, and limited as it has been so far, has not included coverage for the player.
It is quite possible that at the moment recording does not work in players.
This is definitely going to be address asap.

I'll schedule some time next week to see how much needs to be adjusted.

**Comment by Cheeseless** [comment]
any news on this one?

**Comment by jasonm-unity** [comment]
There is a scheduled QA round happeding this week. Based on the results, I'll examine the scope of getting to work on for standalone players. Video, will still be limited to Windows machines.

**Comment by Pimp-Cpp** [comment]
Is it possible to get a status update for this issue?
On my Windows machine, I can record videos from within my game in editor mode, but not from standalone build.

**Comment by jasonm-unity** [comment]
no work has been done yet on that aspect.
The main issue being that video recording is not supported yet, by unity,
on most platforms.

**Comment by keirankozlowski** [comment]
Wanted to again check the status on this. 

I am working on a project where I need to trigger a recording based on an event, similar to a instant replay as in games like Overwatch. Several such events would be triggered per user session and each event would need to produce a separate video file.

Is something like this anywhere near feasible?

**Comment by jasonm-unity** [comment]
as long as you are running it inside the editor that's simple to do.
Create a timeline that records what you want,
The event would then just trigger the timeline to run.

**Comment by keirankozlowski** [comment]
Yeah unfortunately it has to be a standalone build. We're working on a demo for potential clients.

I will keep watching the status of this feature, thanks!

**Comment by caseypugh** [comment]
+1 - Would love if this could work in a standalone build, and not just in the Unity Editor. Opens this up to way more possibilities and in-game capture experiences.

**Comment by RikuTheFuffs** [comment]
+1, would be lovely to be able to do what keirankozlowski and caseypugh are saying.
Moreover, is there a way to enable mouse capturing in editor? ( at the moment if the mouse is visibile in the game it isn't recorded in the video)

**Comment by Darako** [comment]
+1, this can be useful for a lot of things!

**Comment by neciszhang** [comment]
+1, this can be useful for a lot of things!

**Comment by vijayxd** [comment]
im still eagerly waiting for this feature any update news??

**Comment by chaselones** [comment]
I can't believe they haven't implemented this yet!  Come on Unity!

**Comment by vikas-shah** [comment]
Would be good to know where the blockers are and if you could suggest alternative workaround to achieve video recording in standalone builds.

**Comment by chaselones** [comment]
I achieved this in a project using a product from the asset store
